platform_versioned_docs/version-24.1/enterprise/release_notes/enterprise_changelog.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: " Enterprise changelog"
+description: "Seqera Platform Enterprise changelog"
+date: "4 July 2024"
+tags: [changelog]
+---
+
+### 24.1.3 Enterprise - 22 August 2024
+
+- Fix: Disable launch form submit button during form validation
+- Fix: Allow workDir edit during pipeline launch 
+- Fix: Data Explorer preview for Azure Blob Storage files
+- Fix: GCP resource leak 
+- Fix: Remote config read issue
+- Fix: Dependency vulnerabilities 
+- Improvement: Add tag propagation for storage volumes to AWS launch template
+- Improvement: Implement custom H8 2nd-level cache to replace Redisson cache 
+- Improvement: Extend `determineTextFormat` helper function 
```

Comment:
New addition included in `24.1.3-RC1` that wasn't present in `24.1.2`: `nf-launcher` has been [bumped](https://github.com/seqeralabs/platform/commit/97eb7fe5e5c6aaf100d8a4ad43aef65225648885).